src/lib/markdown.js

```diff
@@ -71,6 +71,6 @@ export function formatMessage(text = '') {
   node.innerHTML = marked.parse(sanitizeHTML(textWithSymbol))
 
   const textWithoutHtml = node.textContent || node.innerText || ''
-  const styledText = textWithoutHtml.replace(/↵/g, '<span class="arrow-return">↵</span>')
+  const styledText = textWithoutHtml.replace(/↵/g, '<span class="arrow-return">↵</span>').replace(/\u2028/g, '<span class="arrow-return">↵</span>')
```

Comment:
Let's move `\u2028` to a const (e.g. `LINE_SEPARATOR`), or maybe a helper function, e.g. `normalizeLineSeparator()`, and reuse it in both places. 

1. Create a const/helper in the same file
2. Add an explanation/comment why this is needed